modules/installation-configure-proxy.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-customizations.adoc
+// * installing/installing_aws_user_infra/installing-aws-user-infra.adoc
+// * installing/installing_bare_metal/installing-bare-metal.adoc
+// * installing/installing_vsphere/installing-vsphere.adoc
+// TODO: gcp
+
+ifeval::["{context}" == "installing-bare-metal"]
+:bare-metal:
+endif::[]
+
+[id="installation-configure-proxy_{context}"]
+= Configuring the cluster-wide proxy during installation
+
+Production environments can deny direct access to the Internet and instead have
+an HTTP or HTTPS proxy available. You can configure a new {product-title}
+cluster to use a proxy by configuring the proxy settings in the
+`install-config.yaml` file.
+
+ifdef::bare-metal[]
+[NOTE]
+====
+For bare metal installations, node IP addresses should be assigned from the `install-config.yaml` file's `networking.machineCIDR` field. If they are not, then the node IP addresses must be manually added to the `install-config.yaml` file's `proxy.noProxy` field.
```

Comment:
s/should/must

Reply:
@kalexand-rh So they technically don't have to, so I didn't think that "must" was appropriate here. Because if they aren't, then that's when they have to fill out the noProxy field. Do you have a different suggestion other than "should" that you'd prefer, that isn't must?

Reply:
@bergerhoffer, maybe:
```
For bare metal installations, if you do not assign node IP addresses from the range that is specified in the `networking.machineCIDR` value in the `install-config.yaml` file, you must include them in the `proxy.noProxy` parameter value in the `install-config.yaml` file. 
```